rental.rb

```diff
@@ -0,0 +1,16 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Kindly add getter and setter methods for person and book.

Reply:
Many thanks I will implement all the changes.